src/main/java/org/example/src/units/Unit.java

```diff
@@ -33,23 +31,19 @@ public void getHit(int Damage)
     {
         currentHealth-=Damage;
     }
-    public void giveHit(Unit object){
+    public void giveHit(Unit object) {
         System.out.println(
-                "\t"
-                +this.name
-                +" attack "
-                +object.name
+                "\t" +
+                this.name+
+                " attack "+
+                object.name
```

Comment:
This looks okay, but try to use String.format or System.out.printf next time (this way you wouldn't have to concatenate the strings, only pass parameters as values for placeholders).